usaspending_api/awards/v2/data_layer/orm.py

```diff
@@ -41,6 +41,9 @@ def construct_assistance_response(requested_award_dict):
         return None
     response.update(award)
 
+    # TODO: Add this to the response and update the contract.
```

Comment:
Should this be a TODO? I know you added the ability to get executive compensation for either FPDS or FABS, but are we sure that this will get added to FABS? If not, then I would be worried that this comment might get confused for expected behavior not implemented.

Reply:
The impression I got from talking to @tony-sappe was that we know we want to add it at some point, but I have not seen any requirements for it in the backlog. Tony, can you clarify?

Reply:
@BrianZito did you talk with @ebdabbs about updating the assistance response in this PR instead of the future?

Reply:
@BrianZito please update the assistance response to include executive compensation details.